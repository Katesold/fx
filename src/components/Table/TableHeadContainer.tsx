import { TableHead, TableHeader, TableHeaderRow } from "./styles";

export const TableHeadContainer = ({ row }: { row: string[] }) => {
    return (
        <TableHead>
            <TableHeaderRow>
                {row?.map(cell => {
                    return (
                        <TableHeader>{cell}</TableHeader>
                    )
                })}
            </TableHeaderRow>
        </TableHead>)
};