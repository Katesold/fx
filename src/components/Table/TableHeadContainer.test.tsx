import { TableHeadContainer } from './TableHeadContainer';
import { render } from '@testing-library/react';

describe('TableHeadContainer components are being tested', () => {
    const props = {
        row: ["Beneficiary ID","Payment Currency","Payment Amount","Payment Reference"]
    };
    it('should match the snapshot', () => {
        const { container } = render(<TableHeadContainer {...props} />);
        expect(container).toMatchSnapshot();
    });
});