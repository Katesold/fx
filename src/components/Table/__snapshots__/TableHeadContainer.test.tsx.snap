// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableHeadContainer components are being tested should match the snapshot 1`] = `
<div>
  <thead
    class="sc-dkrFOg fXYAxm"
  >
    <tr
      class="sc-jSUZER kLPgkR"
    >
      <th
        class="sc-eDvSVe BNyIW"
      >
        Beneficiary ID
      </th>
      <th
        class="sc-eDvSVe BNyIW"
      >
        Payment Currency
      </th>
      <th
        class="sc-eDvSVe BNyIW"
      >
        Payment Amount
      </th>
      <th
        class="sc-eDvSVe BNyIW"
      >
        Payment Reference
      </th>
    </tr>
  </thead>
</div>
`;
